### PR TITLE
V-polytope volume example

### PR DESCRIPTION
**Pull request by surajchoubey**
This PR contains VPolytope volume approximation example added.

### CODE REVIEW

**Review by vissarion — COMMENTED** [review]
Thanks for this PR! 

I have a few comments. 

- The current example is very slow. You could improve the runtime by checking different polytope, e.g. `generate_cross<>()` instead of `generate_cube<>()`. Also disactivating the ` volume_sequence_of_balls` algorithm or activate it only in very small instances. We want the whole example to be executed in few secs on an ordinary pc. 
-  Reading a VPolytope from ine file does not seem to work on my side.

**Comment by surajchoubey** [comment]
Well I added a wrong comment in there. 
Correction: 
As per C++ Interface Documentation of volume_approximation,
VPolytopes and Zonotopes meant to be read in *.ext extension
HPolytopes by *.ine extension

I'll be implementing the above changes soon...

File: examples/vpolytope-volume/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# VolEsti (volume computation and sampling library)
```
Comment (vissarion) [inline examples/vpolytope-volume/CMakeLists.txt]: is this file needed?

Comment (surajchoubey) [reply examples/vpolytope-volume/CMakeLists.txt]: Yes it contains some necessary cmake instructions
```add_executable (vpolytopevolume vpolytopevolume.cpp)```
```TARGET_LINK_LIBRARIES(vpolytopevolume ${LP_SOLVE})```

Comment (vissarion) [reply examples/vpolytope-volume/CMakeLists.txt]: but you can use the cmake file from `examples` folder to build this example

Comment (surajchoubey) [reply examples/vpolytope-volume/CMakeLists.txt]: OK implementing now
